playbooks/foreman_proxy_content.yml

```diff
@@ -15,7 +15,7 @@
       foreman_installer_disable_system_checks: true
       foreman_installer_options_internal_use_only:
         - '--certs-tar-file "{{ foreman_proxy_content_certs_tar }}"'
-        - "{{ '' if (foreman_repositories_version == 'nightly' or 2.4 is version(foreman_repositories_version, '>')) else ('--foreman-proxy-content-enable-ostree true') }}"
+        - "{{ '--foreman-proxy-content-enable-ostree true' if (ansible_distribution_major_version == '8' and (foreman_repositories_version == 'nightly' or foreman_repositories_version is version('3.2', '>='))) else '' }}"
```

Comment:
Should we be looking at the future and make it `!= '7'`?

Reply:
maybe